### DISCUSSION TITLE
Android: Readium 2 kotlin: is it possible to print EPUB from  reader page.

### DISCUSSION THREAD

**Discussion by nareshsvn** [readium/kotlin-toolkit#91] [Q&A]
Hi,

Is it possible to print EPUB from the reader page Readium 2 kotlin?  

Thanks,
Naresh

**Comment by mickael-menu** [comment]
No it's not possible and I'm not sure this will ever be on the roadmap.

Soon though you will be able to iterate over the text content of a publication.
